libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxDistanceFormatter.kt

```diff
@@ -33,43 +30,55 @@ import java.util.Locale
  * @param context from which to get localized strings from
  * @param roundingIncrement increment by which to round small distances
  */
-class MapboxDistanceFormatter(
+class MapboxDistanceFormatter private constructor(
```

Comment:
I could remove the private I just thought it better to enforce usage of the builder.

Reply:
Sorry the breaking change is not in this line, it comes because we don't request a `language` (`String`) anymore and a `Locale` is needed instead. Not a problem only noting so we don't forget to include it in our migration guides 🚀